packages/components/src/components/Tabs/Tabs.stories.tsx

```diff
@@ -13,7 +13,7 @@ export default {
   component: Tabs,
   subcomponents: { Tab, TabList, TabPanels, TabPanel },
   title: "Components/Tabs",
-} as ComponentMeta<typeof Tabs>;
+} as unknown as ComponentMeta<typeof Tabs>;
```

Comment:
What's the deal with the unknown here?

Reply:
Since `aria-label` is required, I was getting a TS error. The unkown resets it.

Reply:
Had to actually ts-ignore this one because changing to unknown removes the story.